Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+pipeline {
+    agent {
+        label 'jdk11'
+    }
+
+    stages {
+        stage('Build') {
+            steps {
+                sh './gradlew build -x test'
```

Comment:
test "works" fine when i have implementation vs compile only, though i'm not sure the test is really testing anything

Reply:
it's not but generally you would separate these out.

test failed for me with compileOnly

Reply:
oh you are saying run tests as a different step. Thats cool.

I think the tests work fine once moved to implementation

Reply:
they do yes